### DISCUSSION TITLE
Can I specify a library with `json-rpc`?

### DISCUSSION THREAD

**Discussion by scbarton** [retorquere/zotero-better-bibtex#2384] [Q&A] [answered]
I have two libraries in my Zotero vault, My Library and one Group Library.  There is quite a bit of duplication between the two libraries, and auto-generated citekeys lead to duplicate citekeys for records that appear in both libraries.  

Because of this duplication, when I generate a bibliography with `json-rpc` using the `item.bibliography` method and a list of `citekeys`, two bibliography entries are generated for `citekeys` that match both libraries. I wonder if there is a way to avoid this, perhaps by specifying a library in the call to `json-rpc`?  Alternatively, could I read records from my `bibtex` export, which is only generated from My Library?

**Comment by retorquere** [comment, marked as answer]
Not yet, I'll add that to the API. Would be easiest for me if you open an issue for this.

**Reply by scbarton** [reply]
[Issue opened](https://github.com/retorquere/zotero-better-bibtex/issues/2392), thanks!
